### PR TITLE
Reformat modules

### PR DESCRIPTION
**Pull request by Idonotus**
Standardize module format through flattening
Should not change functionality or features

### CODE REVIEW

File: src/client/scripts/game/chess/movesets.js [hunk 1]
```diff
@@ -6,203 +6,198 @@
...
+                    // + Knight moveset!
+                    [-2,1],[-1,2],[1,2],[2,1],
+                    [-2,-1],[-1,-2],[1,-2],[2,-1]
+                ]
+            };
+        },
+    };
+}
 
-export default movesets;
```
Comment (Naviary3) [inline src/client/scripts/game/chess/movesets.js:208]: Is the Piece type definition exported?

File: src/client/scripts/game/chess/organizedlines.js [hunk 2]
```diff
@@ -24,342 +24,358 @@ import coordutil from '../misc/coordutil.js';
...
+    organizePiece,
+    removeOrganizedPiece,
+    initUndefineds,
+    appendUndefineds,
+    areWeShortOnUndefineds,
+    addMoreUndefineds,
+    insertUndefinedIntoList,
+    buildKeyListFromState,
+    buildStateFromKeyList,
+    getEmptyTypeState,
```
Comment (Naviary3) [inline src/client/scripts/game/chess/organizedlines.js:376]: getEmptyTypeState() will only ever be called from inside this module, so it doesn't need to be exported.

File: src/client/scripts/game/chess/specialmove.js [hunk 3]
```diff
@@ -9,116 +9,121 @@ import coordutil from '../misc/coordutil.js';
...
-
-export default specialmove;
\ No newline at end of file
+export default {
+    getFunctions,
+    kings,
+    pawns,
+    isPawnMoveADoublePush,
+    getEnPassantSquare,
+    getEnpassantCaptureCoords,
```
Comment (Naviary3) [inline src/client/scripts/game/chess/specialmove.js:128]: getFunctions() is the only method called externally, all others don't need to be exported, but kept private.

File: src/client/scripts/game/chess/specialundo.js [hunk 4]
```diff
@@ -10,113 +10,116 @@ import coordutil from '../misc/coordutil.js';
...
-};
+    return true; // Special move has been undo'd!
+}
 
-export default specialundo;
\ No newline at end of file
+export default {
+    getFunctions,
+    kings,
+    pawns,
```
Comment (Naviary3) [inline src/client/scripts/game/chess/specialundo.js:123]: getFunctions() is the only method called externally

File: src/client/scripts/game/gui/stats.js [hunk 5]
```diff
@@ -16,111 +16,125 @@ import config from '../config.js';
...
+    showPiecesMesh,
+    updatePiecesMesh,
+    hidePiecesMesh,
+    showFPS,
+    hideFPS,
+    updateFPS,
+    showRotateMesh,
+    updateRotateMesh,
+    hideRotateMesh,
+    hideMoveLooking
```
Comment (Naviary3) [inline src/client/scripts/game/gui/stats.js:139]: hideMoves() is not called externally, neither:
setTextContentOfMoves,

and hideMoveLooking() is depricated, may as well delete.

**Comment by Naviary3** [comment]
@Idonotus I just considered the possibility that flattened modules might be broken in production mode, as the builder has to compile them all into one script, which may lead to a bunch of duplicate functions. But, maybe the builder is smart enough to avoid that by isolating the module. Have you tested production mode?

**Comment by Idonotus** [comment]
@Naviary3 I did some testing
1) It renames functions to avoid collisions
2) It "simulates" exports to avoid changing everything else

It has been working and i don't foresee this being a problem

**Comment by Naviary2** [comment]
Thanks! Looks good to me! One less level of indentation everywhere will be nice.

**Review by Naviary2 — APPROVED** [review]

**Pull request merged**